clients/privacy-center/middleware.ts

```diff
@@ -0,0 +1,17 @@
+import { NextResponse } from "next/server";
+
```

Comment:
```suggestion
import type { NextRequest} from "next/server";
```
with this you'll be able to type the function below like this `middleware(request: NextRequest)`